kits/hrdf/6-dof_arm.hrdf

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 6-DoF Arm -->
+<robot version="1.1.0">
+    
+  <!-- 6-DoF arm (NOT CURRENTLY same as arm kit) -->  
+  <actuator type="X8-9"/>
+  <bracket type="X5HeavyRightInside"/>
+  <actuator type="X8-16"/>
+  <link type="X5" extension="0.325" twist="pi"/>
+  <actuator type="X8-9"/>
+  <link type="X5" extension="0.325" twist="pi"/>
+  <actuator type="X5-4"/>
+  <bracket type="X5LightRight"/>
+  <actuator type="X5-1"/>
+  <bracket type="X5LightLeft"/>
+  <actuator type="X5-1"/>
+
+  <!-- Add an extra body for the gripper -->
+  <!-- rigid-body mass="0.300" com_trans="0.0 0.0 0.025" output_trans="0.0 0.0 0.075"/ -->
+    
+</robot>
```

Comment:
Same -- I would save this with part number.

....but, why are there two different places where this is saved?  Can we condense?